docs/metaprogramming/01_datatypes.md

```diff
@@ -163,7 +163,7 @@ Lean provides a syntax to quickly convert any Lean expression into an `Expr`.
 
 The process of taking a string of unicode characters and converting them a Lean expression is called __elaboration__. Elaboration is a huge topic that will be covered later ([NOTE] still to do.).
 
-A shorthand for going from an `Expr` `e` to a Lean object is to use `%%e`. This is called  an __anti-quotation__. So for example ```(f $ %%e)`` would create ``expr.app (expr.const `f) e``. [todo] is this still the Lean 4 syntax?
```

Comment:
extra backtick here is correct

Reply:
Okay. I was confused since the Markdown file renders the plain text rather than the code in a monospace font.